src/app/services/validation.service.ts

```diff
@@ -0,0 +1,28 @@
+export class ValidationService {
+  static getValidatorErrorMessage(validatorName: string, validatorValue?: any) {
+    let config = {
```

Comment:
change the name to a meaningful name like "error messages" not config

Reply:
fixed